portfolio/src/main/webapp/main_page_clickables/comment-form.html

```diff
@@ -0,0 +1,9 @@
+<form action="/data" method="POST">
+
```

Comment:
Since you included bootstrap into your page, I just wanted to point out the opinionated bootstrap markup for laying out forms:

https://getbootstrap.com/docs/3.4/css/#forms
https://getbootstrap.com/docs/3.4/css/#forms-horizontal

I'm never *totally* happy with how it looks, but the option is out there for you to try.